BookingsApi/BookingsApi.DAL/Migrations/20220420114450_FixForSENDCaseTypeUserRole.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class FixForSENDCaseTypeUserRole : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            var sqlScript = $@"
+                declare @caseTypeId int;
+                select @caseTypeId = Id from CaseType
+                where Name = 'Special Educational Needs and Disability';
+
+                declare @caseRoleId1 int;
+                select @caseRoleId1 = Id from CaseRole
+                where CaseTypeId = @caseTypeId and Name = 'Appellant';
+
+                declare @caseRoleId2 int;
+                select @caseRoleId2 = Id from CaseRole
+                where CaseTypeId = @caseTypeId and Name = 'Local Authority'
+
+                declare @hearingRoleId1 int;
+                select @hearingRoleId1 = Id from HearingRole
+                where Name = 'MacKenzie Friend' and CaseRoleId = @caseRoleId1
+
+                declare @hearingRoleId2 int;
+                select @hearingRoleId2 = Id from HearingRole
+                where Name = 'Witness' and CaseRoleId = @caseRoleId1
+
+                declare @hearingRoleId3 int;
+                select @hearingRoleId3 = Id from HearingRole
+                where Name = 'Witness' and CaseRoleId = @caseRoleId2
+
+                declare @userRoleId int;
+                select @userRoleId = Id from UserRole where Name = 'Individual';
+
+                update HearingRole set UserRoleId = @userRoleId
+                where Id in (@hearingRoleId1, @hearingRoleId2, @hearingRoleId3);
+                ";
+
+            migrationBuilder.Sql(sqlScript);
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            var sqlScript = @"
```

Comment:
The `Up()` method looks fine but the `Down()` isn't quite the opposite of it. If the Up method updates the hearing roles then the Down method should try to revert those changes.

I think it should be a case of copying the Up method, but changing the `select @userRoleId` line to 'Representative', or whatever it is currently.

Reply:
you are right I will amend it